src/MakieSlides.jl

```diff
@@ -250,8 +253,25 @@ end
 const pygments = PyCall.PyNULL()
 const pygments_lexers = PyCall.PyNULL()
 const pygments_styles = PyCall.PyNULL()
-const EMOJIS = Dict{String,String}()
 const RGX_EMOJI = r":([a-zA-Z0-9_-]+):"
+const EMOJIS_MAP = Dict{String,String}()
+const EMOJIS_PNGPATH = normpath(joinpath(@__DIR__, "..", "assets", "openmoji_png"))
+
+
+function emoji_filename(shorthand::String)
+    !haskey(EMOJIS_MAP, shorthand) && error("Unknown emoji shorthand '$shorthand'")
+    emoji = EMOJIS_MAP[shorthand]
+    unicodes = [ @sprintf "%X" codepoint(c) for c in emoji ]
+    join(unicodes, "-")
+end
+
+emoji_filename_png(shorthand::String) =
+    joinpath(EMOJIS_PNGPATH, emoji_filename(shorthand) * ".png")
+
+function load_emoji_image(filename)
+    img = load(filename)
+    convert.(ARGB{Float32}, img)
```

Comment:
```suggestion
    convert.(Makie.Colors.RGBAf, img)
```
this is the type which doesn't get converted, AFAIK all the rest do.  Should save a little time.